docs/running_on_testnet.md

```diff
@@ -88,13 +88,16 @@ Currency: DAT
 ```
 Note you can only use the faucet once per day. Use the testnet faucet available at https://faucet.vana.org to fund your wallets, or ask a DAT holder to send you some test DAT tokens.
 
-Get your wallet address for all three accounts
+Get your wallet private keys for all three accounts and import them into metamask or another wallet of your choice.
 ```bash
-jq -r '.address' ~/.vana/wallets/owner/hotkeys/default
-jq -r '.address' ~/.vana/wallets/validator_4000/hotkeys/default
-jq -r '.address' ~/.vana/wallets/validator_4001/hotkeys/default
+jq -r '.privateKey' ~/.vana/wallets/owner/coldkey
+jq -r '.privateKey' ~/.vana/wallets/validator_4000/coldkey
```

Comment:
Does this work? I thought the coldkey is encrypted, we can't `jq` it directly.

Reply:
It should only work if you create the coldkey with the `--no_password` option, in which case it gets stored unencrypted.

Reply:
I used `--no_password`, going to update doc to specify that